tools/prepare-template-directory/readme.md

```diff
@@ -0,0 +1,45 @@
+## Preconditions
+* Install Bitnami's Sealed Secrets: ...
+* GOlang: `brew install golang`
+### Prepare the GOlang program
+```shell
+go mod init extract-version
+go mod tidy
+go get gopkg.in/yaml.v3
+```
+
+### Usage
+```shell
+go run tools/prepare-template-directory/prepare-template-directory.go --help
+
+Usage of /var/folders/74/f87zby6s2cn6q_jsn5n6zmj4wjs0zx/T/go-build3319720530/b001/exe/prepare-template-directory:
+  -cas-template-dir string
+        Path to the directory with pom.xml files (optional) (default "template-server-spring")
+  -cas-template-openapi-pom-file string
+        location of the template for the OpenAPI POM (JSON, optional) (default "template-server-spring/openapi/pom.xml")
+  -debug
+        Enable debug output
+  -dry-run
+        don't rewrite pom.xml files
```

Comment:
das Apostroph wird im Code als Textfeld angesehen.

Reply:
Das ist die Wiedergabe einer Meldung, die Bestandteil einer Ausgabe der GOlang Applikation ist.

Wie würdest Du so etwas in ein `readme.md` `reinschreiben? Gerne auch an dieser Stelle vorschlagen:
![image](https://github.com/minova-afis/aero.minova.os.github.workflows/assets/95353437/ec7877f4-18af-431f-9333-441b7c8c9a27)